src/scripts/cf/form-tags/Tag.ts

```diff
@@ -210,17 +272,34 @@ namespace cf {
 			let isValid: boolean = true;
 			let valueText: string = value.text;
 
-			if(this.pattern){
-				isValid = this.pattern.test(valueText);
```

Comment:
HTML5 pattern is a valid way to do validation, maybe instead of removing it do a check if pattern tag exist (then use it) else check for custom validation rules?